framework/wasm-adapter/src/api/blockchain_api_node.rs

```diff
@@ -356,10 +356,9 @@ impl BlockchainApiImpl for VmApiImpl {
         &self,
         token_id_handle: Self::ManagedBufferHandle,
     ) -> multiversx_sc::types::EsdtLocalRoleFlags {
-        unsafe {
-            multiversx_sc::types::EsdtLocalRoleFlags::from_bits_unchecked(getESDTLocalRoles(
-                token_id_handle,
-            ) as u64)
-        }
+        multiversx_sc::types::EsdtLocalRoleFlags::from_bits(unsafe {
+            getESDTLocalRoles(token_id_handle)
+        } as u64)
+        .unwrap()
```

Comment:
Unwrap not allowed here, since it can panic.
We need to find a better solution.
Specifically in this case I think error can be ignored

Reply:
in case of error, call
`error_hook::signal_error(message)`
on else